codegenerator/cli/templates/static/codegen/src/bindings/HyperSyncClient.res

```diff
@@ -428,6 +428,7 @@ let make = (~url) =>
     url,
     enableChecksumAddresses: true,
     bearerToken: Env.envioApiToken->Belt.Option.getWithDefault(defaultToken),
+    httpReqTimeoutMillis: Env.hyperSyncClientTimeoutMillis->Belt.Option.getWithDefault(120_000),
```

Comment:
The default is 30s in the client

Reply:
Should we pass this to the HyperFuel client as well?

Reply:
Possibly, I've never seen this issue TBH until trying this configuration of an indexer. There is something to do with decompressing the transaction input in the client that takes super long. HyperSync should never take longer than 5s per query. The rest of the time is either in the network or in the local operations. A timeout and retry makes sense if it's network latency but definitely needs to be increased if it's deterministically hitting the timeout.